main.go

```diff
@@ -1,39 +1,52 @@
 package main // import "moul.io/moulsay"
 
 import (
+	"context"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"strings"
 
-	cli "gopkg.in/urfave/cli.v2"
+	"github.com/peterbourgon/ff/v3/ffcli"
 	"moul.io/moulsay/moulsay"
 )
 
 func main() {
-	app := cli.App{
-		Flags: []cli.Flag{
-			&cli.IntFlag{Name: "max-width", Aliases: []string{"w"}, Value: 72},
+	fs := flag.NewFlagSet("app", flag.ExitOnError)
+	maxWidth := fs.Int("max-width", 72, "set max width")
+
+	root := &ffcli.Command{
+		Name:       "moul say",
+		ShortUsage: "moulsay + word",
+		ShortHelp:  "moulsay word",
+		LongHelp:   "moulsay -max-width=? word for different max width from moul to word (minimum max-width = 27)",
+		FlagSet:    fs,
+		Exec: func(ctx context.Context, args []string) error {
+			err := run(ctx, args, *maxWidth)
+			if err != nil {
+				return err
+			}
+			return nil
 		},
-		Action: run,
 	}
-	if err := app.Run(os.Args); err != nil {
-		log.Printf("error: %v\n", err)
-		os.Exit(1)
+
+	if err := root.ParseAndRun(context.Background(), os.Args[1:]); err != nil {
+		log.Fatal(err)
 	}
 }
 
-func run(c *cli.Context) error {
-	message := strings.Join(c.Args().Slice(), " ")
+func run(c context.Context, args []string, maxWidth int) error {
```

Comment:
```suggestion
func run(ctx context.Context, args []string, maxWidth int) error {
```

I'm not sure this is mandatory, but since, `ctx context.Context` is special by the fact that it should always be the first argument, please use the ultra well-known form: `ctx` instead of `c`

Reply:
@moul Yes this is mandatory Exec is ->  func(ctx context.Context, args []string) error